ftw/contentstats/demo/filters.py

```diff
@@ -33,3 +34,17 @@ def keep(self, key):
         if key in whitelist:
             return True
         return False
+
+
+@implementer(IStatsKeyFilter)
+@adapter(IPloneSiteRoot, Interface)
+class DiskUsagePathFilter(object):
+
+    def __init__(self, context, request):
+        self.context = context
+        self.request = request
+
+    def keep(self, key):
+        if basename(key).endswith('.tmp'):
```

Comment:
With the current implementation can this ever be true?
The paths are limited to `var/blobstorage` and `var/filestorage/Data.fs` or do I misunderstand something?